index.html

```diff
@@ -12,38 +12,52 @@
   <div>
     {% assign sorted_hacks = site.hacks | where:"happened",false | sort:"date" | reverse %}
     {% assign next_hack = sorted_hacks | first %}
-    {% assign start_time = next_hack.date | slice:0,10 | append:"T09:30Z" %}
+    {% assign start_time = next_hack.date | slice:0,10 | append:"T08:30Z" %}
```

Comment:
I'm assuming there's some way we can make it just use UTC/GMT, and then update as needed on the client-side...but doesn't show this month anyway.

Reply:
yeah, I'm not sure tbh.  We needed #89 to switch it the other way round too 😓